reel-picker/src/main/java/com/innowisegroup/reelpicker/datetime/LocalTime.kt

```diff
@@ -15,13 +15,35 @@ class LocalTime private constructor(val hour: Int, val minute: Int) : Serializab
 
     override fun hashCode(): Int = 31 * hour + minute
 
+    fun plusHours(hoursToAdd: Int): LocalTime =
+        if (hoursToAdd == 0) {
+            this
+        } else {
+            val newHour = hour + hoursToAdd
+            of(newHour, minute)
+        }
+
+    fun minusHours(hoursToSubtract: Int): LocalTime = plusHours(-hoursToSubtract)
+
+    fun plusMinutes(minutesToAdd: Int): LocalTime =
```

Comment:
this implementation differs from java.time. Take a look in source code. Test prove it

Reply:
I made like this (taken from sources)

![image](https://user-images.githubusercontent.com/79689735/186515318-d71cd037-b1cd-45cb-86b1-916ee8a1a22b.png)